llama_stack/providers/tests/safety/fixtures.py

```diff
@@ -27,19 +27,38 @@ def safety_remote() -> ProviderFixture:
     return remote_stack_fixture()
 
 
+def safety_model_from_shield(shield_id):
+    if shield_id in ("Bedrock", "CodeScanner", "CodeShield"):
```

Comment:
Do we want specific non-meta implemented providers hardcoded in our tests?

Reply:
I think `CodeScanner` and `CodeShield` are specific constants used in their impls (and I should use a reference to that to be sure.) We need some way to identifier we are using Bedrock in our test case.